test/util/client.go

```diff
@@ -1,13 +1,23 @@
 package util
 
 import (
+	"os"
+	"path"
 	"path/filepath"
 
+	kclient "k8s.io/kubernetes/pkg/client"
+
 	"github.com/openshift/origin/pkg/client"
 	configapi "github.com/openshift/origin/pkg/cmd/server/api"
-	kclient "k8s.io/kubernetes/pkg/client"
+	cmdutil "github.com/openshift/origin/pkg/cmd/util"
+	"github.com/openshift/origin/pkg/cmd/util/tokencmd"
 )
 
+// GetBaseDir returns the base directory used for test.
+func GetBaseDir() string {
+	return cmdutil.Env("BASETMPDIR", path.Join(os.TempDir(), "openshift-"+Namespace()))
```

Comment:
I think for extended we have /tmp/openshift-extended... in case you want to run integration and extended in parallel.